dask/array/tests/test_array_core.py

```diff
@@ -2589,7 +2589,7 @@ def test_from_func():
 
     assert d.shape == x.shape
     assert d.dtype == x.dtype
-    assert_eq(d.compute(), 2 * x)
+    assert_eq(d, 2 * x)
```

Comment:
let me know if you want me to revert this. It's technically an improvement, but I was really just trying to trigger the label :)